src/components/internals/tag-remove/tag-remove.js

```diff
@@ -2,14 +2,23 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { CloseBoldIcon } from '../../icons';
 
-const TagRemove = props => (
-  <div {...props.innerProps}>
-    <CloseBoldIcon
-      theme={props.selectProps.isDisabled ? 'grey' : 'black'}
-      size="medium"
-    />
-  </div>
-);
+const TagRemove = props => {
+  let innerProps = props.innerProps;
+  if (props.selectProps.isDisabled) {
```

Comment:
if it's disabled, then we don't spread the `onClick, onTouchStart and onTouchEnd` events onto the surrounding div.

Reply:
Can you leave a code comment with that explanation?

Reply:
sure

Reply:
I think [this](https://github.com/commercetools/ui-kit/pull/855/commits/a186362383f1eb7d5edcaa2b79727b203d28d62d) is more clear